src/target/riscv/riscv.c

```diff
@@ -2386,6 +2594,13 @@ static const struct command_registration riscv_exec_command_handlers[] = {
 			"(optional) to indicate Bscan Tunnel Type {0:(default) NESTED_TAP , "
 			"1: DATA_REGISTER}"
 	},
+	{
+		.name = "disable_virt2phys",
```

Comment:
Can you change this from disable to enable, and have it enabled by default? That avoids commands with double negatives like "riscv disable_virt2phy off" which can be hard to understand.

Please also document this new command in doc/openocd.texi.

Reply:
I found there is already a command called 'set_enable_virtual' in the document. I will reuse the command to turn on/off the virtual address translation.